docs/dev/tutorial/android-debugging.md

```diff
@@ -0,0 +1,127 @@
+# Prerequisites
+
+Everything required for this can be installed through the
+prereq\_android\_sdk\_install.sh script in e-mission-phone. The most important
+packages are the emulator, platform-tools, and a system-image for your
+architecture. Use `sdkmanager --list_installed` to check installed android
+packages.
+
+Add `$ANDROID_HOME/emulator` and `$ANDROID_HOME/platform-tools` to your `$PATH`
+if they aren't already added.
+
+# Setting Up the Target
+
+Setup the android emulator or connect a phone over usb/wifi.
+
+## Android Emulator
+
+Note that the android emulator does not simulate the geofence so trips need to
+be started manually in the developer options.
+
+Create an Android Virtual Device:
+
+```
+avdmanager avd create \
+-k 'system-images;android-34;google_apis_playstore;x86_64' \
+-n <avd-name>
+```
```

Comment:
Note that you can use different system images to simulate older or newer versions of android. The system image here should just be a placeholder.